src/components/Tooltip/index.js

```diff
@@ -1,22 +1,27 @@
 import _ from 'underscore';
 import React, {PureComponent} from 'react';
 import {Animated, View} from 'react-native';
+import {BoundsObserver} from '@react-ng/bounds-observer';
 import TooltipRenderedOnPageBody from './TooltipRenderedOnPageBody';
 import Hoverable from '../Hoverable';
 import withWindowDimensions from '../withWindowDimensions';
-import {propTypes, defaultProps} from './tooltipPropTypes';
+import * as tooltipPropTypes from './tooltipPropTypes';
 import TooltipSense from './TooltipSense';
-import makeCancellablePromise from '../../libs/MakeCancellablePromise';
 import * as DeviceCapabilities from '../../libs/DeviceCapabilities';
 
+// A "target" for the tooltip, i.e. an element that, when hovered over, triggers the tooltip to appear. The tooltip will
```

Comment:
This comment confuses me a bit. Is it meant to describe what `class Tooltip` is? If so, I am confused that `class Tooltip` is the target for the tooltip 😕

Reply:
Ok so why is this comment here? It feels out of context, no?

Reply:
Can we put it where we define the target for example?

Reply:
@flodnv But it is exactly how you say it is! This is confusing.

We all know what a tooltip is. And the `Tooltip` component never modeled a tooltip. Check any use. It's used for wrapping things like avatars, etc., which can _have_ a tooltip. It's always been this way.

Some comments and variable names were confusing, because it wasn't obvious if we were talking about the tooltip itself or its target.

The comment is as it should be, from my perspective. The remaining question is if we want to rename the `Tooltip` component app-wise, or maybe expand the comment to "explain ourselves"

Reply:
Yes

Reply:
I don't feel that passionately but I would say :donothing: ... if you look at how this component is used, it's like this:

```js
<Tooltip text="this is a button">
    <Button />
</Tooltip>
```

Which I feel is pretty intuitive. It's true that under the hood we really only have one global tooltip with absolute positioning, and this component just provides an anchor / hook for that tooltip. So maybe `TooltipAnchor` or `TooltipWrapper` would be a more technically accurate name, but personally I think that most of the time developers don't need to care about that and can just think of it as a `Tooltip`.

Reply:
I personally agree that the comment is a bit confusing and that the name is fine. For all intents and purposes it is a... tooltip. The "target" feels like it would refer to the wrapped element. If this comment is attempting to explain how the tooltip works at a high level it could probably be better.

Reply:
@flodnv You're right, this is much better and what I actually had in mind. I'll create a follow-up PR tomorrow.

Reply:
https://github.com/Expensify/App/pull/19519